website/content/ja/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: TAG Environmental Sustainability
+toc_hide: true
+---
+
+<div class="row mt-5 mb-3">
+    <div class="col-lg-6">
+        <div class="lead">
+        TAG Environmental Sustainabilityは、クラウドネイティブアプリケーションのデリバリに関する構築、パッケージング、デプロイ、管理、運用を含むプロジェクトとイニシアチブをサポートします。
+        </div>
+    </div>
+    <div class="col-lg-6">
+        <img src="/images/tag-environmental-sustainability_color.svg" alt="Tag Environmental Sustainability logo" style="max-width: 300px;">
+    </div>
+</div>
+
+<p class="mt-5 mb-5"><img src="/images/tag-env-sustainability-header.webp" alt="TAG Environmental Sustainability"></p>
+
+このテクニカルアドバイザリーグループの目的は、クラウドネイティブテクノロジーにおける環境の持続可能性イニシアチブの開発やサポート、そして評価するのに役立つ提唱をします。このTAGは、クラウドネイティブツールを通じてサービスプロバイダーが消費量やカーボンフットプリントを削減するための価値と可能なインセンティブを特定します。
+
+<!-- cSpell:ignore Linktree -->
+TAG Environmental Sustainabilityの最も重要なリンクは、Linktreeページから利用できます: [cncfenvtag](https://linktr.ee/cncfenvtag).
+
+- [GitHubレポジトリ](https://github.com/cncf/tag-env-sustainability)
```

Comment:
```suggestion
- [GitHubリポジトリ](https://github.com/cncf/tag-env-sustainability)
```

please use "リポジトリ" instead of "レポジトリ"

FYI: https://docs.github.com/ja/repositories/